utils/test_helpers.py

```diff
@@ -61,7 +61,7 @@ def create_user_and_sounds(num_sounds=1, num_packs=0, user=None, count_offset=0,
     Creates User, Sound and Pack objects useful for testing. A 'sound_counter' is used to make sound names unique as
     well as other fields like md5.
     NOTE: creating sounds requires License objects to exist in DB. Do that by making sure your test case loads
-    'initial_data' fixture, i.e. "fixtures = ['initial_data']".
+    'licenses' fixture, i.e. "fixtures = ['licenses']".
     :param num_sounds: N sounds to generate
```

Comment:
not super important, but what I do when I make changes and see a related change is do that one too. You could have updated the docstring format here

Reply:
I agree. I think I changed the format somewhere else but I must have forgotten this one.

Reply:
Addressed inb1ae6e0e09564ede08626f4d0cff25e1610c10d0
It is interesting the case where we return tuples. I found an explanation somewhere about how to deal with that in the docstring, but I think there is no common way to do it. Anyway, if you like the way it has been approached here maybe we could copy this docstring as an example in the developers file.

Reply:
Further formatted and added as example in `DEVELOPERS` 840e6700728e8e9116aea7fdc0fe1ba4e52fb26d